app/src/main/res/values/strings.xml

```diff
@@ -695,11 +695,17 @@
     <string name="sslErrorLeaveCTA">Leave This Site</string>
     <string name="sslErrorAdvancedCTA">Advanced</string>
     <string name="sslErrorExpandedHeadline">DuckDuckGo warns you when a website has an invalid certificate.</string>
-    <string name="sslErrorExpandedCTA"><![CDATA[<u>Accept Risk and Visit Site<u/>]]></string>
+    <string name="sslErrorExpandedCTA"><![CDATA[<u>Accept Risk and Visit Site</u>]]></string>
```

Comment:
We should move all these to donottranslate.xml (create if it doesn't exist) so lint checks don't fail because of missing translations